### PR TITLE
Feature/mydb

### PR DESCRIPTION
**Pull request by assadounto**
### In this feature I tried PostgreSQL
Here in the link to the [Screenshots](https://github.com/assadounto/my_postgresql/tree/feature/mydb/__screenshots__)

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,10 @@
...
-    name varchar(100)
-);
+CREATE TABLE ANIMALS(
+ID INT GENERATED ALWAYS AS IDENTITY,
+name VARCHAR(100),
+DATE_OF_BIRTH DATE,
+ESCAPE_ATTEMPTS INT, 
+WEIGHT_KG DECIMAL
+NEUTERED  BOOLEAN,
+);
```
Comment (danmainah) [inline schema.sql:10]: - [ ] [optional] It would be nice to mark the id column as primary key.

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,11 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM ANIMALS WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
+SELECT * FROM ANIMALS WHERE weight_kg BETWEEN 10.3 AND 17.4;
```
Comment (danmainah) [inline queries.sql]: - [ ] Kindly check your weight figures, the weight should be between 10.4kg and 17.3kg.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @assadounto , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Gitflow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: queries.sql (hunk 2 above)
Comment (assadounto) [reply queries.sql]: Hi @danmainah Please it says it should include that too

**Review by ShahierNashaat — APPROVED** [review]
Hi @assadounto, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Good overall design ✔️
- Your code is clean and well documented ✔️

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by assadounto** [comment]
Thank you @ShahierNashaat and @danmainah

**Pull request merged**